src/app/app-routing.module.ts

```diff
@@ -1,7 +1,27 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomePageComponent } from './modules/home-page/components/home-page/home-page.component';
+import { PointsShopPageComponent } from './modules/pointsshop-page/components/pointsshop-page/pointsshop-page.component';
```

Comment:
Just a typo, I guess. Usually developers add empty line between libraries imports and your files imports. I see you did it in another files.

```suggestion
import { NgModule } from '@angular/core';
import { RouterModule, Routes } from '@angular/router';

import { HomePageComponent } from './modules/home-page/components/home-page/home-page.component';
import { PointsShopPageComponent } from './modules/pointsshop-page/components/pointsshop-page/pointsshop-page.component';
```
On my current projects there are even three various blocks of imports: main technology imports (in our case it's about angular modules), outside libraries and your files.